packages/mui-joy/src/ChipDelete/ChipDeleteProps.ts

```diff
@@ -1,10 +1,34 @@
-import { OverrideProps } from '@mui/types';
 import * as React from 'react';
+import { OverridableStringUnion, OverrideProps } from '@mui/types';
+import { SxProps } from '../styles/defaultTheme';
+import { ColorPaletteProp, VariantProp } from '../styles/types';
 
 export type ChipDeleteSlot = 'root';
 
+export interface ChipPropsColorOverrides {}
```

Comment:
Should we name this `ChipDeletePropsColorOverrides`?

Reply:
Not sure why but it is updated on my end. Can you check it again.

Reply:
Yes, it is updated.